docs/testing.rst

```diff
@@ -68,6 +69,37 @@ The :ref:`Project` object for your project.
         assert 'MyContract' in project.compiled_contract_data
 
 
+How populus finds the project of the project fixture
+''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+If no other argument is provided, populus assumes that the tested project directory is the one where the tests run from.
+This is true if you run py.test from within the project's directory, or with a positional argument to this projecr's directory,
+e.g. ``$ py.test /path/to/my/project/``.
+
+If the tests are in a different directory, e.g. ``$ py.tests /path/to/tests/``,
+you will have to provide the tested project:
+
+1. With command line argument: ``$ py.test /path/to/tests/ --populus-project /path/to/my/project/``
+2. Or, in a pytest.ini file, with the following entry: ``populus_project=/path/to/my/project/``
+3. Or with an environment variable: ``PYTEST_POPULUS_PROJECT``. E.g., ``$ export PYTEST_POPULUS_PROJECT=/path/to/my/project/``
+
+If a project is provided in more than one place, then the first is the command line, then pytest.ini, then the environment variable.
```

Comment:
I think this would be more clear as:

The precedence order for these different methods of setting the project directory is:

#. command line
#. pytest.ini
#. environment variable

(Also, does it make since to change that to have environment var take precedence over pytest.ini?)